### ISSUE TITLE
SEO

### ISSUE THREAD

**Issue by shadinamrouti** [aspnet/Docs#10370]
For SEO purposes, I suggest you use &lt;h1&gt; instead of &lt;h2&gt; in the replacement of the new view content. Thanks. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 788d46c9-a91a-2788-8db7-b53c37b512f2
* Version Independent ID: 00c2c01d-d235-2d2d-5c96-1f33a9314382
* Content: [Add a view to an ASP.NET Core MVC app](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/adding-view?view=aspnetcore-2.2&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/first-mvc-app/adding-view.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/tutorials/first-mvc-app/adding-view.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by shadinamrouti** [comment]
Hi, I'm talking about the tutorial content itself, it says and I quote: 

"Replace the contents of the Views/HelloWorld/Index.cshtml Razor view file with the following HTML: 

@{
    ViewData["Title"] = "Index";
}

<h2>Index</h2>

<p>Hello from our View Template!</p>"

My suggestion is to use <h1>index</h1> because teaching people to start a page with an <h2> header is bad. They should start their HTML with a <h1> tag "at least and at most". Thank you.

**Comment by guardrex** [comment]
Yes, and that's the way they built the templates in the last several versions. I've remarked on that to them, too :point_right: https://github.com/aspnet/Templating/issues/67 (3rd entry)

In the latest version of the template, they changed it, so I think it's good now. They currently use ...

```cshtml
@{
    ViewData["Title"] = "Home Page";
}

<div class="text-center">
    <h1 class="display-4">Welcome</h1>
    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
</div>
```

Since you're referring to the topic content, I'll re-open so that @Rick-Anderson can take a look when he gets online.
